Sources/BottomSheetCalculator.swift

```diff
@@ -30,68 +30,82 @@ struct BottomSheetCalculator {
         return max(superview.frame.height - makeTargetHeight(), handleHeight)
     }
 
-    /// Calculates bottom and top thresholds for the given target offsets.
+    /// Creates the layout of the BottomSheetView based on the target offsets and the current target offset
```

Comment:
Perhaps `layout` is a bit misleading here in the docs.

Reply:
Yes, forgot to update this one